lib/tower_rollbar/rollbar/item.ex

```diff
@@ -177,4 +177,30 @@ defmodule TowerRollbar.Rollbar.Item do
   defp os do
     "type: #{inspect(:os.type())} version: #{inspect(:os.version())}"
   end
+
+  def json_prepare(map) when is_map(map) do
```

Comment:
This `is_map` is also true for structs and it was not the intention.

It eventually fails  with 

```
(Protocol.UndefinedError) protocol Enumerable not implemented for ... (a struct)
```